openrtb_ext/imp_deepintent.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ExtImpDeepintent struct {
+	TagId string `json:tagId`
```

Comment:
Nitpick: Should be TagID per Go naming conventions.

Reply:
Changed